test/spec/aggregate_verify.test.ts

```diff
@@ -13,6 +13,7 @@ interface IAggregateSigsVerifyTestCase {
     };
     output: boolean;
   };
+  meta?: undefined;
```

Comment:
there are still a bunch of these, can you please double check the diff of this PR and remove any change that is not strictly required

Reply:
Should be good now.